isacc_messaging/api/email_notifications.py

```diff
@@ -54,6 +58,109 @@ def send_message_received_notification(recipients: list, patient: Patient):
     )
 
 
+def assemble_outgoing_counts_email(practitioner, patients):
+    """Pull together email content for given practitioner and their list of patients
+
+    :param practitioner: Practitioner object, target of email
+    :param patients: list of Patient objects for whom the practitioner is assigned,
+      expected to only include those with an outgoing message in the next 24 hours
+    :return: email content
+    """
+    SUPPORT_EMAIL = current_app.config.get('ISACC_SUPPORT_EMAIL')
+    UNSUB_LINK = f'{current_app.config.get("ISACC_APP_URL")}/unsubscribe'
+    patient_list_url = f'{current_app.config.get("ISACC_APP_URL")}/home?sort_by=next_message&sort_direction=desc&flags=following'
+    primary, secondary = [], []
+    for p in patients:
+        if p.generalPractitioner and p.generalPractitioner[0].reference == f"Practitioner/{practitioner.id}":
+            primary.append(p)
+            continue
+        secondary.append(p)
+
+    subject = f"ISACC {len(patients)} Caring Contact sending today"
+    contents = []
+    if primary:
+        contents.append(f"Caring Contact messages will be sent to {len(patients)} of your recipients today.")
+    if secondary:
+        contents.append(f"{len(primary)} of those contacts for whom you are the primary author,")
+        contents.append(f"{len(secondary)} of those contacts for whom you are the secondary author.")
+    msg = " ".join(contents)
```

Comment:
@mcjustin I extended the spec with the info above in the `secondary` clause, as it was easy to generate and I know Bill cares to communicate this.  Let me know if you don't want this included.

Reply:
@pbugni  I don't see edits to [the spec](https://docs.google.com/document/d/1jDFFoO7nMQr_6qZIHicZr1rp_RcvK8wpn89IMDjiO24/edit#bookmark=id.jjmk8mxgb6cf)... it already included the "secondary" count, ack I edited some weeks ago and left it in suggestion mode. I'd like to stick w/ the spec's verbiage - "secondary" author is not a term that I've heard the ISACC team use. Are you saying that Bill suggested the term "secondary"?

Reply:
@pbugni  also, I'm inclined to always include the count "for whom you are following". Users get accustomed to quick pattern matching so I'd rather that the verbiage was consistent.
```
Today Caring Contact messages will be sent to [n2] recipients for whom you are the primary author, and [n3] for whom you are following. 
...
```

Reply:
i probably invented the `secondary` term, will use your suggestion.  i didn't see the comments in suggestion mode, thank you (@mcjustin) for the clarity i now see in the spec.

Reply:
The latest email (18:36 today) looks good to me, thanks @pbugni